mig-frankfurt/dataelementhub.gui
# #110 - Allow adding customized text and logos to the homepage

**What's in the PR**
* Allow adding customized text and logos to the homepage
* close #110

**How to test manually**
1. Go to assets -> others -> fill the homepage.md file out
2. Go to assets -> images -> logo -> create "extern" folder -> add all logos that you would like to display on the homepage
3. Start the program
4. Check if the text and logos look as expected
